app/src/main/res/layout/activity_program_mode.xml

```diff
@@ -1,32 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/hill_background"
     tools:context=".ProgramModeActivity">
 
-    <RelativeLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
+    <include layout="@layout/layout_focus_screen" />
```

Comment:
@CoderVishalSehgal I think we should make different program mode activities rather than just toggling the visibilities of them (their layouts) in a single parent activity layout. @iamareebjamal What do you think?

Reply:
Use Fragment